<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Smilei</class>
 <widget class="QMainWindow" name="Smilei">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>601</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SmileiQt</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QComboBox" name="timeStep"/>
    </item>
    <item row="1" column="0" colspan="3">
     <widget class="QWidget" name="widget" native="true">
      <layout class="QGridLayout" name="gridLayout_3" rowstretch="1,0,0" columnstretch="1,1">
       <item row="0" column="0">
        <layout class="QGridLayout" name="grid1"/>
       </item>
       <item row="0" column="1">
        <layout class="QGridLayout" name="grid2"/>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="field2"/>
       </item>
       <item row="2" column="0">
        <widget class="QComboBox" name="field1"/>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QWidget" name="widget" native="true">
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="margin">
        <number>0</number>
       </property>
       <item row="0" column="4">
        <widget class="QCheckBox" name="logBox">
         <property name="text">
          <string>Log</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Max</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="mini">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Min</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QLineEdit" name="maxi">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>530</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_File"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionChange_Dir">
   <property name="text">
    <string>Open File</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionOpen_File">
   <property name="text">
    <string>Open File</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>mini</tabstop>
  <tabstop>maxi</tabstop>
  <tabstop>logBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
